telegraf build fails on SmartOS (illumos)

## Bug report

### Relevant telegraf.conf:

### System info:

Telegraf version: v1.4 (according to CHANGELOG.md)
operating system: SmartOS base-64 2017Q1 image
Go: go version go1.8 solaris/amd64

### Steps to reproduce:

    GOPATH=${PWD}
    PATH=${GOPATH}/bin:${PATH}
    go get golang.org/x/sys/unix
    go get github.com/sparrc/gdm
    go get github.com/influxdata/telegraf
    make -C src/github.com/influxdata/telegraf

### Expected behavior:

Successful build

### Actual behavior:

    go install -ldflags \
            "-X main.version=1.2.0-rc1-245-g4c534433 -X main.commit=4c534433 -X main.branch=master" ./...
    ../tail/watch/inotify.go:13:2: no buildable Go source files in /root/smartos-tick/src/github.com/influxdata/tail/vendor/gopkg.in/fsnotify.v1
    Makefile:18: recipe for target 'build' failed
    make[1]: *** [build] Error 1
    make[1]: Leaving directory '/root/smartos-tick/src/github.com/influxdata/telegraf'
    Makefile:16: recipe for target 'bin/telegraf' failed
    make: *** [bin/telegraf] Error 2

That doesn't seem to work for me.

<details>

<!-- -->

```
[root@gobuild ~/smartos-tick]# export GOPATH=${PWD}
[root@gobuild ~/smartos-tick]# export PATH=${GOPATH}/bin:${PATH}
[root@gobuild ~/smartos-tick]# go get golang.org/x/sys/unix
[root@gobuild ~/smartos-tick]# go get github.com/sparrc/gdm
[root@gobuild ~/smartos-tick]# go get github.com/influxdata/telegraf
[root@gobuild ~/smartos-tick]# GOOS=solaris GOARCH=amd64 gmake -C src/github.com/influxdata/telegraf telegraf
gmake: Entering directory '/root/smartos-tick/src/github.com/influxdata/telegraf'
go build -o telegraf -ldflags "-X main.commit=73897d1f -X main.branch=master" ./cmd/telegraf/telegraf.go
plugins/parsers/collectd/parser.go:9:2: cannot find package "collectd.org/api" in any of:
        /opt/local/go/src/collectd.org/api (from $GOROOT)
        /root/smartos-tick/src/collectd.org/api (from $GOPATH)
plugins/parsers/collectd/parser.go:10:2: cannot find package "collectd.org/network" in any of:
        /opt/local/go/src/collectd.org/network (from $GOROOT)
        /root/smartos-tick/src/collectd.org/network (from $GOPATH)
plugins/inputs/kafka_consumer/kafka_consumer.go:14:2: cannot find package "github.com/Shopify/sarama" in any of:
        /opt/local/go/src/github.com/Shopify/sarama (from $GOROOT)
        /root/smartos-tick/src/github.com/Shopify/sarama (from $GOPATH)
plugins/inputs/aerospike/aerospike.go:15:2: cannot find package "github.com/aerospike/aerospike-client-go" in any of:
        /opt/local/go/src/github.com/aerospike/aerospike-client-go (from $GOROOT)
        /root/smartos-tick/src/github.com/aerospike/aerospike-client-go (from $GOPATH)
plugins/outputs/riemann/riemann.go:12:2: cannot find package "github.com/amir/raidman" in any of:
        /opt/local/go/src/github.com/amir/raidman (from $GOROOT)
        /root/smartos-tick/src/github.com/amir/raidman (from $GOPATH)
plugins/inputs/zipkin/handler.go:10:2: cannot find package "github.com/apache/thrift/lib/go/thrift" in any of:
        /opt/local/go/src/github.com/apache/thrift/lib/go/thrift (from $GOROOT)
        /root/smartos-tick/src/github.com/apache/thrift/lib/go/thrift (from $GOPATH)
plugins/inputs/cloudwatch/cloudwatch.go:9:2: cannot find package "github.com/aws/aws-sdk-go/aws" in any of:
        /opt/local/go/src/github.com/aws/aws-sdk-go/aws (from $GOROOT)
        /root/smartos-tick/src/github.com/aws/aws-sdk-go/aws (from $GOPATH)
internal/config/aws/credentials.go:5:2: cannot find package "github.com/aws/aws-sdk-go/aws/client" in any of:
        /opt/local/go/src/github.com/aws/aws-sdk-go/aws/client (from $GOROOT)
        /root/smartos-tick/src/github.com/aws/aws-sdk-go/aws/client (from $GOPATH)
internal/config/aws/credentials.go:6:2: cannot find package "github.com/aws/aws-sdk-go/aws/credentials" in any of:
        /opt/local/go/src/github.com/aws/aws-sdk-go/aws/credentials (from $GOROOT)
        /root/smartos-tick/src/github.com/aws/aws-sdk-go/aws/credentials (from $GOPATH)
internal/config/aws/credentials.go:7:2: cannot find package "github.com/aws/aws-sdk-go/aws/credentials/stscreds" in any of:
        /opt/local/go/src/github.com/aws/aws-sdk-go/aws/credentials/stscreds (from $GOROOT)
        /root/smartos-tick/src/github.com/aws/aws-sdk-go/aws/credentials/stscreds (from $GOPATH)
internal/config/aws/credentials.go:8:2: cannot find package "github.com/aws/aws-sdk-go/aws/session" in any of:
        /opt/local/go/src/github.com/aws/aws-sdk-go/aws/session (from $GOROOT)
        /root/smartos-tick/src/github.com/aws/aws-sdk-go/aws/session (from $GOPATH)
plugins/inputs/cloudwatch/cloudwatch.go:11:2: cannot find package "github.com/aws/aws-sdk-go/service/cloudwatch" in any of:
        /opt/local/go/src/github.com/aws/aws-sdk-go/service/cloudwatch (from $GOROOT)
        /root/smartos-tick/src/github.com/aws/aws-sdk-go/service/cloudwatch (from $GOPATH)
plugins/outputs/kinesis/kinesis.go:9:2: cannot find package "github.com/aws/aws-sdk-go/service/kinesis" in any of:
        /opt/local/go/src/github.com/aws/aws-sdk-go/service/kinesis (from $GOROOT)
        /root/smartos-tick/src/github.com/aws/aws-sdk-go/service/kinesis (from $GOPATH)
plugins/inputs/kafka_consumer/kafka_consumer.go:15:2: cannot find package "github.com/bsm/sarama-cluster" in any of:
        /opt/local/go/src/github.com/bsm/sarama-cluster (from $GOROOT)
        /root/smartos-tick/src/github.com/bsm/sarama-cluster (from $GOPATH)
plugins/inputs/couchbase/couchbase.go:7:2: cannot find package "github.com/couchbase/go-couchbase" in any of:
        /opt/local/go/src/github.com/couchbase/go-couchbase (from $GOROOT)
        /root/smartos-tick/src/github.com/couchbase/go-couchbase (from $GOPATH)
plugins/inputs/docker/client.go:8:2: cannot find package "github.com/docker/docker/api/types" in any of:
        /opt/local/go/src/github.com/docker/docker/api/types (from $GOROOT)
        /root/smartos-tick/src/github.com/docker/docker/api/types (from $GOPATH)
plugins/inputs/docker/docker_testdata.go:9:2: cannot find package "github.com/docker/docker/api/types/container" in any of:
        /opt/local/go/src/github.com/docker/docker/api/types/container (from $GOROOT)
        /root/smartos-tick/src/github.com/docker/docker/api/types/container (from $GOPATH)
plugins/inputs/docker/docker_testdata.go:10:2: cannot find package "github.com/docker/docker/api/types/registry" in any of:
        /opt/local/go/src/github.com/docker/docker/api/types/registry (from $GOROOT)
        /root/smartos-tick/src/github.com/docker/docker/api/types/registry (from $GOPATH)
plugins/inputs/docker/client.go:9:2: cannot find package "github.com/docker/docker/client" in any of:
        /opt/local/go/src/github.com/docker/docker/client (from $GOROOT)
        /root/smartos-tick/src/github.com/docker/docker/client (from $GOPATH)
plugins/inputs/docker/client.go:10:2: cannot find package "github.com/docker/go-connections/sockets" in any of:
        /opt/local/go/src/github.com/docker/go-connections/sockets (from $GOROOT)
        /root/smartos-tick/src/github.com/docker/go-connections/sockets (from $GOPATH)
plugins/inputs/mqtt_consumer/mqtt_consumer.go:15:2: cannot find package "github.com/eclipse/paho.mqtt.golang" in any of:
        /opt/local/go/src/github.com/eclipse/paho.mqtt.golang (from $GOROOT)
        /root/smartos-tick/src/github.com/eclipse/paho.mqtt.golang (from $GOPATH)
plugins/inputs/mysql/mysql.go:17:2: cannot find package "github.com/go-sql-driver/mysql" in any of:
        /opt/local/go/src/github.com/go-sql-driver/mysql (from $GOROOT)
        /root/smartos-tick/src/github.com/go-sql-driver/mysql (from $GOPATH)
filter/filter.go:6:2: cannot find package "github.com/gobwas/glob" in any of:
        /opt/local/go/src/github.com/gobwas/glob (from $GOROOT)
        /root/smartos-tick/src/github.com/gobwas/glob (from $GOPATH)
plugins/inputs/webhooks/webhooks.go:9:2: cannot find package "github.com/gorilla/mux" in any of:
        /opt/local/go/src/github.com/gorilla/mux (from $GOROOT)
        /root/smartos-tick/src/github.com/gorilla/mux (from $GOPATH)
plugins/inputs/consul/consul.go:6:2: cannot find package "github.com/hashicorp/consul/api" in any of:
        /opt/local/go/src/github.com/hashicorp/consul/api (from $GOROOT)
        /root/smartos-tick/src/github.com/hashicorp/consul/api (from $GOPATH)
plugins/inputs/logparser/logparser.go:10:2: cannot find package "github.com/influxdata/tail" in any of:
        /opt/local/go/src/github.com/influxdata/tail (from $GOROOT)
        /root/smartos-tick/src/github.com/influxdata/tail (from $GOPATH)
internal/config/config.go:29:2: cannot find package "github.com/influxdata/toml" in any of:
        /opt/local/go/src/github.com/influxdata/toml (from $GOROOT)
        /root/smartos-tick/src/github.com/influxdata/toml (from $GOPATH)
internal/config/config.go:30:2: cannot find package "github.com/influxdata/toml/ast" in any of:
        /opt/local/go/src/github.com/influxdata/toml/ast (from $GOROOT)
        /root/smartos-tick/src/github.com/influxdata/toml/ast (from $GOPATH)
logger/logger.go:10:2: cannot find package "github.com/influxdata/wlog" in any of:
        /opt/local/go/src/github.com/influxdata/wlog (from $GOROOT)
        /root/smartos-tick/src/github.com/influxdata/wlog (from $GOPATH)
plugins/inputs/postgresql/postgresql.go:12:2: cannot find package "github.com/jackc/pgx/stdlib" in any of:
        /opt/local/go/src/github.com/jackc/pgx/stdlib (from $GOROOT)
        /root/smartos-tick/src/github.com/jackc/pgx/stdlib (from $GOPATH)
cmd/telegraf/telegraf.go:24:2: cannot find package "github.com/kardianos/service" in any of:
        /opt/local/go/src/github.com/kardianos/service (from $GOROOT)
        /root/smartos-tick/src/github.com/kardianos/service (from $GOPATH)
plugins/inputs/exec/exec.go:14:2: cannot find package "github.com/kballard/go-shellquote" in any of:
        /opt/local/go/src/github.com/kballard/go-shellquote (from $GOROOT)
        /root/smartos-tick/src/github.com/kballard/go-shellquote (from $GOPATH)
plugins/inputs/prometheus/parser.go:19:2: cannot find package "github.com/matttproud/golang_protobuf_extensions/pbutil" in any of:
        /opt/local/go/src/github.com/matttproud/golang_protobuf_extensions/pbutil (from $GOROOT)
        /root/smartos-tick/src/github.com/matttproud/golang_protobuf_extensions/pbutil (from $GOPATH)
plugins/inputs/dns_query/dns_query.go:10:2: cannot find package "github.com/miekg/dns" in any of:
        /opt/local/go/src/github.com/miekg/dns (from $GOROOT)
        /root/smartos-tick/src/github.com/miekg/dns (from $GOPATH)
plugins/inputs/nats_consumer/nats_consumer.go:11:2: cannot find package "github.com/nats-io/nats" in any of:
        /opt/local/go/src/github.com/nats-io/nats (from $GOROOT)
        /root/smartos-tick/src/github.com/nats-io/nats (from $GOPATH)
plugins/inputs/nsq_consumer/nsq_consumer.go:9:2: cannot find package "github.com/nsqio/go-nsq" in any of:
        /opt/local/go/src/github.com/nsqio/go-nsq (from $GOROOT)
        /root/smartos-tick/src/github.com/nsqio/go-nsq (from $GOPATH)
plugins/inputs/zipkin/convert.go:11:2: cannot find package "github.com/openzipkin/zipkin-go-opentracing/_thrift/gen-go/zipkincore" in any of:
        /opt/local/go/src/github.com/openzipkin/zipkin-go-opentracing/_thrift/gen-go/zipkincore (from $GOROOT)
        /root/smartos-tick/src/github.com/openzipkin/zipkin-go-opentracing/_thrift/gen-go/zipkincore (from $GOPATH)
plugins/outputs/prometheus_client/prometheus_client.go:17:2: cannot find package "github.com/prometheus/client_golang/prometheus" in any of:
        /opt/local/go/src/github.com/prometheus/client_golang/prometheus (from $GOROOT)
        /root/smartos-tick/src/github.com/prometheus/client_golang/prometheus (from $GOPATH)
plugins/inputs/prometheus/parser.go:20:2: cannot find package "github.com/prometheus/client_model/go" in any of:
        /opt/local/go/src/github.com/prometheus/client_model/go (from $GOROOT)
        /root/smartos-tick/src/github.com/prometheus/client_model/go (from $GOPATH)
plugins/inputs/prometheus/parser.go:21:2: cannot find package "github.com/prometheus/common/expfmt" in any of:
        /opt/local/go/src/github.com/prometheus/common/expfmt (from $GOROOT)
        /root/smartos-tick/src/github.com/prometheus/common/expfmt (from $GOPATH)
plugins/outputs/kinesis/kinesis.go:10:2: cannot find package "github.com/satori/go.uuid" in any of:
        /opt/local/go/src/github.com/satori/go.uuid (from $GOROOT)
        /root/smartos-tick/src/github.com/satori/go.uuid (from $GOPATH)
plugins/inputs/procstat/process.go:7:2: cannot find package "github.com/shirou/gopsutil/cpu" in any of:
        /opt/local/go/src/github.com/shirou/gopsutil/cpu (from $GOROOT)
        /root/smartos-tick/src/github.com/shirou/gopsutil/cpu (from $GOPATH)
plugins/inputs/system/mock_PS.go:9:2: cannot find package "github.com/shirou/gopsutil/disk" in any of:
        /opt/local/go/src/github.com/shirou/gopsutil/disk (from $GOROOT)
        /root/smartos-tick/src/github.com/shirou/gopsutil/disk (from $GOPATH)
plugins/inputs/system/system.go:9:2: cannot find package "github.com/shirou/gopsutil/host" in any of:
        /opt/local/go/src/github.com/shirou/gopsutil/host (from $GOROOT)
        /root/smartos-tick/src/github.com/shirou/gopsutil/host (from $GOPATH)
plugins/inputs/system/mock_PS.go:11:2: cannot find package "github.com/shirou/gopsutil/load" in any of:
        /opt/local/go/src/github.com/shirou/gopsutil/load (from $GOROOT)
        /root/smartos-tick/src/github.com/shirou/gopsutil/load (from $GOPATH)
plugins/inputs/system/mock_PS.go:12:2: cannot find package "github.com/shirou/gopsutil/mem" in any of:
        /opt/local/go/src/github.com/shirou/gopsutil/mem (from $GOROOT)
        /root/smartos-tick/src/github.com/shirou/gopsutil/mem (from $GOPATH)
plugins/inputs/system/mock_PS.go:13:2: cannot find package "github.com/shirou/gopsutil/net" in any of:
        /opt/local/go/src/github.com/shirou/gopsutil/net (from $GOROOT)
        /root/smartos-tick/src/github.com/shirou/gopsutil/net (from $GOPATH)
plugins/inputs/procstat/process.go:8:2: cannot find package "github.com/shirou/gopsutil/process" in any of:
        /opt/local/go/src/github.com/shirou/gopsutil/process (from $GOROOT)
        /root/smartos-tick/src/github.com/shirou/gopsutil/process (from $GOPATH)
plugins/inputs/snmp/snmp.go:18:2: cannot find package "github.com/soniah/gosnmp" in any of:
        /opt/local/go/src/github.com/soniah/gosnmp (from $GOROOT)
        /root/smartos-tick/src/github.com/soniah/gosnmp (from $GOPATH)
plugins/inputs/amqp_consumer/amqp_consumer.go:10:2: cannot find package "github.com/streadway/amqp" in any of:
        /opt/local/go/src/github.com/streadway/amqp (from $GOROOT)
        /root/smartos-tick/src/github.com/streadway/amqp (from $GOPATH)
plugins/inputs/mock_Plugin.go:6:2: cannot find package "github.com/stretchr/testify/mock" in any of:
        /opt/local/go/src/github.com/stretchr/testify/mock (from $GOROOT)
        /root/smartos-tick/src/github.com/stretchr/testify/mock (from $GOPATH)
plugins/inputs/logparser/grok/grok.go:13:2: cannot find package "github.com/vjeantet/grok" in any of:
        /opt/local/go/src/github.com/vjeantet/grok (from $GOROOT)
        /root/smartos-tick/src/github.com/vjeantet/grok (from $GOPATH)
plugins/inputs/kafka_consumer_legacy/kafka_consumer_legacy.go:14:2: cannot find package "github.com/wvanbergen/kafka/consumergroup" in any of:
        /opt/local/go/src/github.com/wvanbergen/kafka/consumergroup (from $GOROOT)
        /root/smartos-tick/src/github.com/wvanbergen/kafka/consumergroup (from $GOPATH)
plugins/inputs/sqlserver/sqlserver.go:11:2: cannot find package "github.com/zensqlmonitor/go-mssqldb" in any of:
        /opt/local/go/src/github.com/zensqlmonitor/go-mssqldb (from $GOROOT)
        /root/smartos-tick/src/github.com/zensqlmonitor/go-mssqldb (from $GOPATH)
plugins/inputs/passenger/passenger.go:13:2: cannot find package "golang.org/x/net/html/charset" in any of:
        /opt/local/go/src/golang.org/x/net/html/charset (from $GOROOT)
        /root/smartos-tick/src/golang.org/x/net/html/charset (from $GOPATH)
plugins/inputs/rethinkdb/rethinkdb.go:11:2: cannot find package "gopkg.in/gorethink/gorethink.v3" in any of:
        /opt/local/go/src/gopkg.in/gorethink/gorethink.v3 (from $GOROOT)
        /root/smartos-tick/src/gopkg.in/gorethink/gorethink.v3 (from $GOPATH)
plugins/inputs/openldap/openldap.go:8:2: cannot find package "gopkg.in/ldap.v2" in any of:
        /opt/local/go/src/gopkg.in/ldap.v2 (from $GOROOT)
        /root/smartos-tick/src/gopkg.in/ldap.v2 (from $GOPATH)
plugins/inputs/mongodb/mongodb.go:17:2: cannot find package "gopkg.in/mgo.v2" in any of:
        /opt/local/go/src/gopkg.in/mgo.v2 (from $GOROOT)
        /root/smartos-tick/src/gopkg.in/mgo.v2 (from $GOPATH)
plugins/inputs/mongodb/mongodb_server.go:10:2: cannot find package "gopkg.in/mgo.v2/bson" in any of:
        /opt/local/go/src/gopkg.in/mgo.v2/bson (from $GOROOT)
        /root/smartos-tick/src/gopkg.in/mgo.v2/bson (from $GOPATH)
plugins/outputs/elasticsearch/elasticsearch.go:14:2: cannot find package "gopkg.in/olivere/elastic.v5" in any of:
        /opt/local/go/src/gopkg.in/olivere/elastic.v5 (from $GOROOT)
        /root/smartos-tick/src/gopkg.in/olivere/elastic.v5 (from $GOPATH)
plugins/inputs/puppetagent/puppetagent.go:5:2: cannot find package "gopkg.in/yaml.v2" in any of:
        /opt/local/go/src/gopkg.in/yaml.v2 (from $GOROOT)
        /root/smartos-tick/src/gopkg.in/yaml.v2 (from $GOPATH)
Makefile:28: recipe for target 'telegraf' failed
gmake: *** [telegraf] Error 1
gmake: Leaving directory '/root/smartos-tick/src/github.com/influxdata/telegraf'
[root@gobuild ~/smartos-tick]# GOOS=solaris GOARCH=amd64 gmake -C src/github.com/influxdata/telegraf 
gmake: Entering directory '/root/smartos-tick/src/github.com/influxdata/telegraf'
gmake deps
gmake[1]: Entering directory '/root/smartos-tick/src/github.com/influxdata/telegraf'
go get github.com/sparrc/gdm
gdm restore
======= Go Dependency Manager =======
= working dir: /root/smartos-tick/src/github.com/influxdata/telegraf
= GOPATH:      /root/smartos-tick
=====================================
> Restoring /root/smartos-tick/src/collectd.org to 2ce144541b8903101fb8f1483cc0497a68798122
> Restoring /root/smartos-tick/src/github.com/aerospike/aerospike-client-go to 95e1ad7791bdbca44707fedbb29be42024900d9c
> Restoring /root/smartos-tick/src/github.com/amir/raidman to c74861fe6a7bb8ede0a010ce4485bdbb4fc4c985
> Restoring /root/smartos-tick/src/github.com/apache/thrift to 4aaa92ece8503a6da9bc6701604f69acf2b99d07
> Restoring /root/smartos-tick/src/github.com/aws/aws-sdk-go to c861d27d0304a79f727e9a8a4e2ac1e74602fdc0
> Restoring /root/smartos-tick/src/github.com/beorn7/perks to 4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9
> Restoring /root/smartos-tick/src/github.com/bsm/sarama-cluster to ccdc0803695fbce22f1706d04ded46cd518fd832
> Restoring /root/smartos-tick/src/github.com/cenkalti/backoff to b02f2bbce11d7ea6b97f282ef1771b0fe2f65ef3
> Restoring /root/smartos-tick/src/github.com/couchbase/go-couchbase to bfe555a140d53dc1adf390f1a1d4b0fd4ceadb28
> Restoring /root/smartos-tick/src/github.com/couchbase/gomemcached to 4a25d2f4e1dea9ea7dd76dfd943407abf9b07d29
> Restoring /root/smartos-tick/src/github.com/couchbase/goutils to 5823a0cbaaa9008406021dc5daf80125ea30bba6
> Restoring /root/smartos-tick/src/github.com/davecgh/go-spew to 346938d642f2ec3594ed81d874461961cd0faa76
> Restoring /root/smartos-tick/src/github.com/docker/docker to f5ec1e2936dcbe7b5001c2b817188b095c700c27
> Restoring /root/smartos-tick/src/github.com/docker/go-connections to 990a1a1a70b0da4c4cb70e117971a4f0babfbf1a
> Restoring /root/smartos-tick/src/github.com/eapache/go-resiliency to b86b1ec0dd4209a588dc1285cdd471e73525c0b3
> Restoring /root/smartos-tick/src/github.com/eapache/go-xerial-snappy to bb955e01b9346ac19dc29eb16586c90ded99a98c
> Restoring /root/smartos-tick/src/github.com/eapache/queue to 44cc805cf13205b55f69e14bcb69867d1ae92f98
> Restoring /root/smartos-tick/src/github.com/eclipse/paho.mqtt.golang to d4f545eb108a2d19f9b1a735689dbfb719bc21fb
> Restoring /root/smartos-tick/src/github.com/go-logfmt/logfmt to 390ab7935ee28ec6b286364bba9b4dd6410cb3d5
> Restoring /root/smartos-tick/src/github.com/go-sql-driver/mysql to 2e00b5cd70399450106cec6431c2e2ce3cae5034
> Restoring /root/smartos-tick/src/github.com/gobwas/glob to bea32b9cd2d6f55753d94a28e959b13f0244797a
> Restoring /root/smartos-tick/src/github.com/go-ini/ini to 9144852efba7c4daf409943ee90767da62d55438
> Restoring /root/smartos-tick/src/github.com/gogo/protobuf to 7b6c6391c4ff245962047fc1e2c6e08b1cdfa0e8
> Restoring /root/smartos-tick/src/github.com/golang/protobuf to 8ee79997227bf9b34611aee7946ae64735e6fd93
> Restoring /root/smartos-tick/src/github.com/golang/snappy to 7db9049039a047d955fe8c19b83c8ff5abd765c7
> Restoring /root/smartos-tick/src/github.com/go-ole/go-ole to be49f7c07711fcb603cff39e1de7c67926dc0ba7
> Restoring /root/smartos-tick/src/github.com/google/go-cmp to f94e52cad91c65a63acc1e75d4be223ea22e99bc
> Restoring /root/smartos-tick/src/github.com/gorilla/mux to 392c28fe23e1c45ddba891b0320b3b5df220beea
> Restoring /root/smartos-tick/src/github.com/hailocab/go-hostpool to e80d13ce29ede4452c43dea11e79b9bc8a15b478
> Restoring /root/smartos-tick/src/github.com/hashicorp/consul to 63d2fc68239b996096a1c55a0d4b400ea4c2583f
> Restoring /root/smartos-tick/src/github.com/influxdata/tail to a395bf99fe07c233f41fba0735fa2b13b58588ea
> Restoring /root/smartos-tick/src/github.com/influxdata/toml to 5d1d907f22ead1cd47adde17ceec5bda9cacaf8f
> Restoring /root/smartos-tick/src/github.com/influxdata/wlog to 7c63b0a71ef8300adc255344d275e10e5c3a71ec
> Restoring /root/smartos-tick/src/github.com/jackc/pgx to b84338d7d62598f75859b2b146d830b22f1b9ec8
> Restoring /root/smartos-tick/src/github.com/jmespath/go-jmespath to bd40a432e4c76585ef6b72d3fd96fb9b6dc7b68d
> Restoring /root/smartos-tick/src/github.com/kardianos/osext to c2c54e542fb797ad986b31721e1baedf214ca413
> Restoring /root/smartos-tick/src/github.com/kardianos/service to 6d3a0ee7d3425d9d835debc51a0ca1ffa28f4893
> Restoring /root/smartos-tick/src/github.com/kballard/go-shellquote to d8ec1a69a250a17bb0e419c386eac1f3711dc142
> Restoring /root/smartos-tick/src/github.com/matttproud/golang_protobuf_extensions to c12348ce28de40eed0136aa2b644d0ee0650e56c
> Restoring /root/smartos-tick/src/github.com/Microsoft/go-winio to ce2922f643c8fd76b46cadc7f404a06282678b34
> Restoring /root/smartos-tick/src/github.com/miekg/dns to 99f84ae56e75126dd77e5de4fae2ea034a468ca1
> Restoring /root/smartos-tick/src/github.com/naoina/go-stringutil to 6b638e95a32d0c1131db0e7fe83775cbea4a0d0b
> Restoring /root/smartos-tick/src/github.com/nats-io/go-nats to ea9585611a4ab58a205b9b125ebd74c389a6b898
> Restoring /root/smartos-tick/src/github.com/nats-io/nats to ea9585611a4ab58a205b9b125ebd74c389a6b898
> Restoring /root/smartos-tick/src/github.com/nats-io/nuid to 289cccf02c178dc782430d534e3c1f5b72af807f
> Restoring /root/smartos-tick/src/github.com/nsqio/go-nsq to a53d495e81424aaf7a7665a9d32a97715c40e953
> Restoring /root/smartos-tick/src/github.com/opencontainers/runc to 89ab7f2ccc1e45ddf6485eaa802c35dcf321dfc8
> Restoring /root/smartos-tick/src/github.com/opentracing-contrib/go-observer to a52f2342449246d5bcc273e65cbdcfa5f7d6c63c
> Restoring /root/smartos-tick/src/github.com/opentracing/opentracing-go to 06f47b42c792fef2796e9681353e1d908c417827
> Restoring /root/smartos-tick/src/github.com/openzipkin/zipkin-go-opentracing to 1cafbdfde94fbf2b373534764e0863aa3bd0bf7b
> Restoring /root/smartos-tick/src/github.com/pierrec/lz4 to 5c9560bfa9ace2bf86080bf40d46b34ae44604df
> Restoring /root/smartos-tick/src/github.com/pierrec/xxHash to 5a004441f897722c627870a981d02b29924215fa
> Restoring /root/smartos-tick/src/github.com/pkg/errors to 645ef00459ed84a119197bfb8d8205042c6df63d
> Restoring /root/smartos-tick/src/github.com/pmezard/go-difflib/difflib to 792786c7400a136282c1664665ae0a8db921c6c2
> Restoring /root/smartos-tick/src/github.com/prometheus/client_golang to c317fb74746eac4fc65fe3909195f4cf67c5562a
> Restoring /root/smartos-tick/src/github.com/prometheus/client_model to fa8ad6fec33561be4280a8f0514318c79d7f6cb6
> Restoring /root/smartos-tick/src/github.com/prometheus/common to dd2f054febf4a6c00f2343686efb775948a8bff4
> Restoring /root/smartos-tick/src/github.com/prometheus/procfs to 1878d9fbb537119d24b21ca07effd591627cd160
> Restoring /root/smartos-tick/src/github.com/rcrowley/go-metrics to 1f30fe9094a513ce4c700b9a54458bbb0c96996c
> Restoring /root/smartos-tick/src/github.com/samuel/go-zookeeper to 1d7be4effb13d2d908342d349d71a284a7542693
> Restoring /root/smartos-tick/src/github.com/satori/go.uuid to 5bf94b69c6b68ee1b541973bb8e1144db23a194b
> Restoring /root/smartos-tick/src/github.com/shirou/gopsutil to 9a4a9167ad3b4355dbf1c2c7a0f5f0d3fb1e9ab9
> Restoring /root/smartos-tick/src/github.com/shirou/w32 to 3c9377fc6748f222729a8270fe2775d149a249ad
> Restoring /root/smartos-tick/src/github.com/Shopify/sarama to c01858abb625b73a3af51d0798e4ad42c8147093
> Restoring /root/smartos-tick/src/github.com/Sirupsen/logrus to 61e43dc76f7ee59a82bdf3d71033dc12bea4c77d
> Restoring /root/smartos-tick/src/github.com/soniah/gosnmp to 5ad50dc75ab389f8a1c9f8a67d3a1cd85f67ed15
> Restoring /root/smartos-tick/src/github.com/StackExchange/wmi to f3e2bae1e0cb5aef83e319133eabfee30013a4a5
> Restoring /root/smartos-tick/src/github.com/streadway/amqp to 63795daa9a446c920826655f26ba31c81c860fd6
> Restoring /root/smartos-tick/src/github.com/stretchr/objx to 1a9d0bb9f541897e62256577b352fdbc1fb4fd94
> Restoring /root/smartos-tick/src/github.com/stretchr/testify to 4d4bfba8f1d1027c4fdbe371823030df51419987
> Restoring /root/smartos-tick/src/github.com/vjeantet/grok to d73e972b60935c7fec0b4ffbc904ed39ecaf7efe
> Restoring /root/smartos-tick/src/github.com/wvanbergen/kafka to bc265fedb9ff5b5c5d3c0fdcef4a819b3523d3ee
> Restoring /root/smartos-tick/src/github.com/wvanbergen/kazoo-go to 968957352185472eacb69215fa3dbfcfdbac1096
> Restoring /root/smartos-tick/src/github.com/yuin/gopher-lua to 66c871e454fcf10251c61bf8eff02d0978cae75a
> Restoring /root/smartos-tick/src/github.com/zensqlmonitor/go-mssqldb to ffe5510c6fa5e15e6d983210ab501c815b56b363
> Restoring /root/smartos-tick/src/golang.org/x/crypto to dc137beb6cce2043eb6b5f223ab8bf51c32459f4
> Restoring /root/smartos-tick/src/golang.org/x/net to f2499483f923065a842d38eb4c7f1927e6fc6e6d
> Restoring /root/smartos-tick/src/golang.org/x/sys to 739734461d1c916b6c72a63d7efda2b27edb369f
> Restoring /root/smartos-tick/src/golang.org/x/text to 506f9d5c962f284575e88337e7d9296d27e729d3
> Restoring /root/smartos-tick/src/gopkg.in/asn1-ber.v1 to 4e86f4367175e39f69d9358a5f17b4dda270378d
> Restoring /root/smartos-tick/src/gopkg.in/fatih/pool.v2 to 6e328e67893eb46323ad06f0e92cb9536babbabc
> Restoring /root/smartos-tick/src/gopkg.in/fsnotify.v1 to a8a77c9133d2d6fd8334f3260d06f60e8d80a5fb
> Restoring /root/smartos-tick/src/gopkg.in/gorethink/gorethink.v3 to 7ab832f7b65573104a555d84a27992ae9ea1f659
> Restoring /root/smartos-tick/src/gopkg.in/ldap.v2 to 8168ee085ee43257585e50c6441aadf54ecb2c9f
> Restoring /root/smartos-tick/src/gopkg.in/mgo.v2 to 3f83fa5005286a7fe593b055f0d7771a7dce4655
> Restoring /root/smartos-tick/src/gopkg.in/olivere/elastic.v5 to 3113f9b9ad37509fe5f8a0e5e91c96fdc4435e26
> Restoring /root/smartos-tick/src/gopkg.in/tomb.v1 to dd632973f1e7218eb1089048e0798ec9ae7dceb8
> Restoring /root/smartos-tick/src/gopkg.in/yaml.v2 to 4c78c975fe7c825c6d1466c42be594d1d6f3aba6
gmake[1]: Leaving directory '/root/smartos-tick/src/github.com/influxdata/telegraf'
gmake telegraf
gmake[1]: Entering directory '/root/smartos-tick/src/github.com/influxdata/telegraf'
go build -o telegraf -ldflags "-X main.commit=73897d1f -X main.branch=master" ./cmd/telegraf/telegraf.go
../tail/watch/inotify.go:13:2: no buildable Go source files in /root/smartos-tick/src/github.com/influxdata/tail/vendor/gopkg.in/fsnotify.v1
Makefile:28: recipe for target 'telegraf' failed
gmake[1]: *** [telegraf] Error 1
gmake[1]: Leaving directory '/root/smartos-tick/src/github.com/influxdata/telegraf'
Makefile:20: recipe for target 'all' failed
gmake: *** [all] Error 2
gmake: Leaving directory '/root/smartos-tick/src/github.com/influxdata/telegraf'
```

<!-- -->

</details>

--- a/plugins/inputs/logparser/logparser.go
+++ b/plugins/inputs/logparser/logparser.go
@@ -1,3 +1,5 @@
+// +build !solaris
+
 package logparser
 
 import (

--- a/plugins/inputs/logparser/logparser_solaris.go
+++ b/plugins/inputs/logparser/logparser_solaris.go
@@ -1,0 +1,3 @@
+// +build solaris
+
+package logparser

--- a/plugins/inputs/tail/tail.go
+++ b/plugins/inputs/tail/tail.go
@@ -1,3 +1,5 @@
+// +build !solaris
+
 package tail
 
 import (

--- a/plugins/inputs/tail/tail_solaris.go
+++ b/plugins/inputs/tail/tail_solaris.go
@@ -1,0 +1,5 @@
+// Skipping plugin on Solaris due to fsnotify support
+//
+// +build solaris
+
+package tail

